src/v0/destinations/braze/transform.js

```diff
@@ -166,6 +166,7 @@ function getUserAttributesObject(message, mappingJson, destination) {
   // reserved keys : already mapped through mappingJson
   const reservedKeys = [
     'address',
+    'home_city',
```

Comment:
`home_city` is braze attribute name - I see `reservedKeys` are according to RS convention, will `home_city` ever come in the payload to consider it as reservedKey ?

Reply:
Removed as it was redundant